pages/homepage/homepage.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="stylesheet" href="https://component-vishalg8454.netlify.app/components/main.css">
+    <link rel="stylesheet" href="./homepage.css">
+    <script src="https://kit.fontawesome.com/06bd83eb3a.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="grid-wrapper">
+
+        <nav class="navbar">
+            <div class="nav-container">
+                <a class="nav-logo" href="#">E-COMMERCE</a>
+                <div class="nav-search-container">
+                    <input class="nav-search-text" type="text">
+                    <button class="nav-search-button"><i class="fas fa-search"></i>
+                    </button>
+                </div>
+                <button class="btn btn-primary-outline  nav-cta">Login</button>
+                <ul class="nav-list">
+                    <li class="nav-list-items">
+                        <div>
+                            <a> <i class="badge-icon fas fa-heart nav-icon"></i></a>
+                            <div class="badge-text">9</div>
+                        </div>
+                    </li>
+                    <li class="nav-list-items">
+                        <div>
+                            <a><i class="badge-icon fas fa-shopping-cart nav-icon"></i></a>
+                            <div class="badge-text">3</div>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+        <div class="carousel">
+            <div class="carousel-container">
+                <img class="carousel-img" src="/assets/FICTION.png">
+                <img class="carousel-img" src="/assets/SELF-HELP.png">
+                <img class="carousel-img" src="/assets/COMIC.png">
+                <img class="carousel-img" src="/assets/SCI-FI.png">
+                <img class="carousel-img" src="/assets/BIOGRAPHY.png">
+                <img class="carousel-img" src="/assets/NOVEL.png">
+            </div>
+        </div>
```

Comment:
Oh, What can be a proper name?

Reply:
since this section categories, you can just name it as category-cards.
also instead of allowing user to scroll horizontally, suggest you to either decrease size of each image so they fit in the page else you can use 'flex-wrap:wrap' so that the images wrap to the next line.